src/state/nftMarket/helpers.ts

```diff
@@ -133,21 +134,27 @@ export const getNftsFromCollectionSg = async (collectionAddress: string): Promis
   }
 }
 
-export const getNftsMarketData = async (where = {}): Promise<NftTokenSg[]> => {
+export const getNftsMarketData = async (
+  where = {},
+  first = 1000,
```

Comment:
Do we have a pagination strategy...? 

Maybe this is one for a conversation tomorrow: "How do we deal with thousands of tokens being on sale."?